sdc/tests/test_dataframe.py

```diff
@@ -810,6 +810,19 @@ def test_impl(df):
                            'D': [None, 'dd', '', None]})
         pd.testing.assert_frame_equal(hpat_func(df), test_impl(df))
 
+    def test_df_isna(self):
+        def test_impl(df):
+            return df.isna()
+        sdc_func = sdc.jit(test_impl)
+        for idx in [[3, 4, 2, 6, 1], ['a', 'b', 'c', 'd', 'e'], None]:
+            df = pd.DataFrame({"FLOAT": [3.2, np.nan, 7.0, 3.3, np.nan],
```

Comment:
Please avoid using uppercase words/sentences where they are not necessary (they put additional emphasis from text formatting perspective). If you want to highlight that each column has it's own dtype in the name of a column just use lowercase names.